app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
please add the FIXME relations, when they will be used, e.g in UI. Remove the FIXMEs for now